### PR TITLE
Fix module extension point

### PR DESCRIPTION
**Pull request by fatmali**

### CODE REVIEW

**Comment by HerbertYiga** [comment]
kindly follow the tips of making pull requests here https://wiki.openmrs.org/display/docs/Pull+Request+Tips

File: omod/src/main/webapp/master-single-page-application.jsp [hunk 1]
```diff
@@ -23,9 +23,13 @@
       System.import("@openmrs/esm-root-config");
       System.import("@openmrs/esm-styleguide");
     </script>
-    <c:import url="${requestScope.spaHeadContentUrl}" />
+        <c:if test="${requestScope.spaHeadContentUrl != null }">
```
Comment (internettrans) [inline omod/src/main/webapp/master-single-page-application.jsp]:
````suggestion
    <c:if test="${requestScope.spaHeadContentUrl != null }">

File: omod/src/main/webapp/master-single-page-application.jsp [hunk 2]
```diff
@@ -23,9 +23,13 @@
...
+        <c:import url="${requestScope.spaHeadContentUrl}" />
```
Comment (internettrans) [inline omod/src/main/webapp/master-single-page-application.jsp]:
```suggestion
      <c:import url="${requestScope.spaHeadContentUrl}" />

File: omod/src/main/webapp/master-single-page-application.jsp [hunk 3]
```diff
@@ -23,9 +23,13 @@
...
+        </c:if>
```
Comment (internettrans) [inline omod/src/main/webapp/master-single-page-application.jsp]:
```suggestion
    </c:if>

File: omod/src/main/webapp/master-single-page-application.jsp [hunk 4]
```diff
@@ -23,9 +23,13 @@
...
   </head>
   <body>
+    <c:if test="${requestScope.spaBodyContentUrl != null }">
     <c:import url="${requestScope.spaBodyContentUrl}" />
```
Comment (internettrans) [inline omod/src/main/webapp/master-single-page-application.jsp]:
```suggestion
      <c:import url="${requestScope.spaBodyContentUrl}" />

**Review by internettrans — CHANGES_REQUESTED** [review]
This looks good. See related https://talk.openmrs.org/t/optional-module-extension-points/24270/2 for the reason behind this change.

**Review by internettrans — APPROVED** [review]
This lgtm

**Pull request merged**